features/dd-sdk-android-webview/src/main/kotlin/com/datadog/android/webview/WebViewTracking.kt

```diff
@@ -137,6 +141,12 @@ object WebViewTracking {
     }
 
     private fun resolveRumFeature(sdkCore: FeatureSdkCore): WebViewRumFeature? {
+        (
+        sdkCore.getFeature(WebViewRumFeature.WEB_RUM_FEATURE_NAME)
+                ?.unwrap<StorageBackedFeature>() as? WebViewRumFeature
```

Comment:
Is there a particular reason for doing `?.unwrap<StorageBackedFeature>() as? WebViewRumFeature` instead of the `?.unwrap<WebViewRumFeature>()`? Same for the similar calls below.

Reply:
oh...nope I think I did that just because I've seen it that way in another place. Better how you suggested

Reply:
I found the explanation why, is because in the core we are force casting it, we are not using `as?` :
```
    override fun <T : Feature> unwrap(): T = wrappedFeature as T
```

Reply:
but it is us who is owning implementation anyway, so we if got the scope for the feature, we can safely unwrap with using `as` under the hood.